common/src/autogluon/common/__init__.py

```diff
@@ -1,6 +1,7 @@
 from .version import __version__
 
 from .utils.log_utils import _add_stream_handler, fix_logging_if_kaggle as __fix_logging_if_kaggle
+from .space import Space, Categorical, Real, Int, Bool
```

Comment:
Shall we remove these imports here since we want users to do `from autogluon.common.space import ...`?